### PR TITLE
feat: dashdirect feature and button on homepage

### PR DESCRIPTION
**Pull request by saratranberg**
Will add the content on wp.dash.org once the css is updated on the server. :)

### CODE REVIEW

File: assets/dash.css [hunk 1]
```diff
@@ -2338,3 +2338,31 @@ margin-bottom: 35px;
...
+    background-position-y: -120px;
+  }
+  .home-banner-btn{
+    margin-top: 10px;
+    max-width: 290px;
+  }
+  .banner .caption h1{
+    padding: 0 40px;
+}
+}
```
Comment (strophy) [inline assets/dash.css:2368]: This requires adding HTML code to the content area of Wordpress, and therefore breaks the clean separation rule of content and layout. You need to abstract the intent of the changes and add them to the theme itself as features, rather than having custom CSS for specific content and putting corresponding HTML in the content. I know it is easier to do it this way, but over time it will become unmaintainable.

File: assets/dash.css [hunk 2]
```diff
@@ -681,8 +681,8 @@ header .lang .dropdown .wpml-ls a {
   .halfbanner .carousel .image {
     background-size: cover;
     background-position: center;
-    height: 285px;
-    width: 100%;
+    height: 340px;
+    width: 95%;
```
Comment (strophy) [inline assets/dash.css:685]: What does this change achieve? I think it is related to the news/blog page, right? In that case it should be in a separate PR.

Comment (saratranberg) [reply assets/dash.css:685]: I left a comment with the change but I don't know where it went. I tried to make it separate but it auto merged it with the other open pull request. Haha! This changes the proportions of the featured blog post image so that it is closer to the dimensions of the featured post images so it doesn't crop them awkwardly.

File: assets/dash.css (hunk 1 above)
Comment (saratranberg) [reply assets/dash.css:2368]: I remember briefly discussing this process with you a while back, but I might need a refresher. Maybe we could set up a meeting so I can have some support my first time adding a theme option?

File: assets/dash.css (hunk 2 above)
Comment (strophy) [reply assets/dash.css:685]: Can you revert this change and put it in a separate PR with an appropriate commit message please :+1:

**Review by strophy — APPROVED** [review]
LGTM! 👍
